src/pages/home/Home.js

```diff
@@ -18,63 +17,79 @@ import config from 'config/constants';
 
 const heroImage = require('./assets/images/coding-coach-logo.svg');
 
-function Home({ t }) {
-  const emailLink = (
-    <a target="_blank" href={`mailto:${config.contact.EMAIL}`} className={styles.email}>
-      {config.contact.EMAIL}
-    </a>
-  );
-  const slackOrg = (
-    <a href={config.contact.SLACK_URL} className={styles.slackOrg}>
-      {t('slack-org')}
-    </a>
-  );
+class Home extends React.Component {
+  render() {
+    const { t } = this.props;
 
-  return (
-    <Fragment>
-      <Navbar />
-      <section className={styles.hero}>
-        <div className={styles.heroInner}>
-          <header className={styles.heroTextWrapper}>
-            <h1 className={styles.heroTitle}>
-              <span>{t('home-header-title')}</span>
-            </h1>
-            <h2 className={styles.heroSubtitle}>{t('home-header-subtitle')}</h2>
-            <Button onClick={noop} className={styles.callToAction}>
-              {t('home-header-cta')}
-            </Button>
-          </header>
-          <Image className={styles.heroImage} alt={t('home-header-media-alt')} src={heroImage} />
-        </div>
-      </section>
+    return (
+      <Fragment>
+        <Navbar />
+        <section className={styles.hero}>
+          <div className={styles.heroInner}>
+            <header className={styles.heroTextWrapper}>
+              <h1 className={styles.heroTitle}>
+                <span>{t('home-header-title')}</span>
+              </h1>
+              <h2 className={styles.heroSubtitle}>{t('home-header-subtitle')}</h2>
+              <Button onClick={() => null} className={styles.callToAction}>
```

Comment:
Please use `utils/noop` to set an empty function

Reply:
oh, I deleted thinking it was a bad practice hahaha ok 🤣